omnidotdev/zenpetal
# Brand colors

## Description

##### Task link: https://trello.com/c/19cz9G1X/35-improve-brand-colors

- Updated brand colors
- Added default color token to each color scale
- Added `Link` component
- Added styles for indeterminate checkbox state

## Test Steps

- Verify `Link` works well (stories useful for baseline)
- Verify updated colors look good
